helm/ako/values.yaml

```diff
@@ -60,11 +60,12 @@ resources:
     cpu: 250m
     memory: 300Mi
   requests:
-    cpu: 100m
-    memory: 75Mi
+    cpu: 250m
+    memory: 200Mi
 
-podSecurityContext: {}
```

Comment:
Can we keep the field podSecurityContext ? In some cases, this may be required.

Reply:
re-instated. thx.